gnome-initial-setup/pages/account/gis-account-page.c

```diff
@@ -1403,6 +1403,9 @@ gis_account_page_init (GisAccountPage *page)
 void
 gis_prepare_account_page (GisDriver *driver)
 {
+  if (gis_driver_is_skipped_page (driver, PAGE_ID))
+    return;
```

Comment:
Hmm.. I tried that, but it does not work as expected: while it does skip the desired page (language, in my case) when clicking on "Next", the "dots indicator" at the bottom does still reflect that there is something in there, because I can see how it moves from the first dot to the third one when skipping over the page I don't want to show.

I think it might be better to leave it as I have it now, then...